packages/genesys-spark-tokens/snapshot-new/json/gse-flare-dark-core.json

```diff
@@ -159,16 +159,16 @@
       "navy": "#203b73"
     },
     "data_visualization": {
-      "yellow_green": "#ddd933",
-      "olive_green": "#868c1e",
-      "aqua_green": "#1da8b3",
-      "pink": "#ff8fdd",
-      "fuchsia": "#cc3ebe",
-      "lilac": "#b5b5eb",
-      "plum": "#5e5782",
-      "electric_purple": "#8452cf",
-      "genesys_blue": "#75a8ff",
-      "navy": "#203b73"
+      "yellow_green": "#f8f3c6",
+      "olive_green": "#dcdbbb",
+      "aqua_green": "#c7e5e8",
+      "pink": "#ffdff5",
+      "fuchsia": "#f4c9ec",
+      "lilac": "#e9e8f9",
+      "plum": "#cdc9d8",
+      "electric_purple": "#dcc9f2",
+      "genesys_blue": "#d9e4ff",
+      "navy": "#bbbfd4"
```

Comment:
These are much lighter versions of the current colours. Just confirming that was intended?

Reply:
99% sure this change is correct. Will be able to confirm 100% once we start testing with UX